src/main/java/com/aws/greengrass/device/DeviceAuthClient.java

```diff
@@ -165,13 +170,14 @@ public void closeSession(String sessionId) throws AuthorizationException {
      */
     @SuppressWarnings("PMD.AvoidUncheckedExceptionsInSignatures")
     public void attachThing(String sessionId, String thingName) throws AuthenticationException {
-        logger.atDebug()
+        logger.atInfo()
                 .kv("sessionId", sessionId)
                 .kv("thingName", thingName)
                 .log("Attaching thing to session");
 
         // Workaround for bridge component
         if (ALLOW_ALL_SESSION.equals(sessionId)) {
+            logger.atInfo().log("Skip attach thing action for the greengrass component");
```

Comment:
Is there value in this log? We have connection / client ID logs from the broker and the corresponding session ID.

Reply:
yeah, broker side should log it. But want to keep CDA side log comprehensive as well.